app/services/topic_sign_service.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "./lib/sql/status"
+
+class TopicSignService
+  attr_reader :search, :results
```

Comment:
suggestion: it doesn't look like either of these are used outside this class, so I would drop the public `attr_reader`